### ISSUE TITLE
OS version not detected for mobile devices

### ISSUE THREAD

**Issue by mkb13** [erossini/BlazorBrowserDetect#6]
The OS version is detected on a Windows PC but not on any of my mobile devices.  On my Pixel 6a (Android),  https://browserdetect.puresourcecode.com/, "Operating System Version" is blank.  Same with my iPad and iPhone.

**Comment by erossini** [comment]
Can you try to refresh the page? Also, can you send me the screenshot of the developer tools?

**Comment by mkb13** [comment]
Yes, refreshed many times.  What do you mean by a screenshot of the developer tools?  This only happens on mobile devices and I'm not sure how to access the developer tools.

**Comment by erossini** [comment]
Depends on the browser, but if you connect your device to your computer, it is possible to open the developer tools from your computer to inspect what is going on the mobile device.

**Comment by noprofoundname2** [comment]
actually it gets detected correctly, although for mobile phones the callback OSVersionUpdate is not triggered. 
In most cases the correct OS version is available through the BrowserInfo. So i stick with that for now.
